src/Umbraco.Web/Routing/PublishedContentRequestEngine.cs

```diff
@@ -606,9 +607,8 @@ private void FindTemplate()
 			// only if the published content is the initial once, else the alternate template
 			// does not apply
             // + optionnally, apply the alternate template on internal redirects
-            var useAltTemplate = _webRoutingSection.DisableAlternativeTemplates == false 
-                && (_pcr.IsInitialPublishedContent
-                || (_webRoutingSection.InternalRedirectPreservesTemplate && _pcr.IsInternalRedirectPublishedContent));
+            var useAltTemplate = _pcr.IsInitialPublishedContent

```

Comment:
Why was `_webRoutingSection.DisableAlternativeTemplates == false` removed here? Seems like this would allow altTemplates again, even when they're completely turned off?